liferay-echo/liferay-portal
# LPS-178958 Convert ResourceCommands into StrutsActions

Motivation
=======
The motivation behind this PR is to improve layouts-reports-web module with new dev patterns.
[LPS-178958](https://issues.liferay.com/browse/LPS-178958)
[LPS-178957](https://issues.liferay.com/browse/LPS-178957)

Proposed Solution
========
What we did to complete this ticket is to migrate portlet resource commands to struts because the portlet is useless since it doesn't contain any action. In order to do that we've remove the LayoutsReportsPortlet, change the structure of the resourceCommands to struts and change the form in which the url are created.

Steps to Verify
========
1. Connect to Page Audit -> https://liferay.atlassian.net/wiki/spaces/ENGTANGO/pages/1586299258/How+to+test+the+Page+Audit+panel+in+localhost
2. Check the panel opens and closes correctly.
3. Check that the information is displayed correctly and click on refresh button the information loads again.

<img width="734" alt="Screenshot 2023-04-21 at 14 24 26" src="https://user-images.githubusercontent.com/91207948/233634787-150481c3-c620-4b0e-9548-47b549699833.png">
